src/cmockery.c

```diff
@@ -1377,6 +1379,22 @@ void* _test_calloc(const size_t number_of_elements, const size_t size,
     return ptr;
 }
 
+//Chech if the address is really allocated
+int check_allocated_exist(void* const ptr){
+    const ListNode * const head = get_allocated_blocks_list();
+    const ListNode *node = head->next;
+    int rc = 0;
+    while(node != head){
```

Comment:
We roughly follow google C++ style https://google.github.io/styleguide/cppguide.html in here.

So this should have some whitespace...

"while (node != head) {"